src/inputs/TextFieldBase.tsx

```diff
@@ -176,6 +178,12 @@ export function TextFieldBase<X extends Only<TextFieldXss, X>>(props: TextFieldB
     e.target.select();
   }, onFocus);
 
+  // Simulate clicking `ElementType` when using an unfocused placeholder
+  function handleUnfocusedPlaceholderClick(e: React.MouseEvent<HTMLDivElement>) {
+    e.stopPropagation();
+    fieldRef.current?.click();
+  }
```

Comment:
**question:** Did you try simulating the `focus` event instead of `click`? I don't know if there's a big difference in the approach, but I kind of felt like simulating focus would've followed at least my mental model a little more. I.e. "User clicks X and we want to move focus to Y". Rather than "User clicks X, but let's pretend they clicked Y". Something about simulating a click feels odd to me, but probably fine. In my mind, the "just move focus" approach aligns better with how a label would interact with a form field, - when a user clicks a label, the field is focused.
Curious if there was a reason to go for the `click` over `focus`.

Reply:
Yes, when we used `focus()` it doesn't open the popup on first click. I didnt dive too deep into the component tree to find what makes `buttonProps` special regarding the onClick though 🤔